### PR TITLE
clean up notation and fix typos

### PR DESCRIPTION
**Pull request by paul-buerkner**
I have been reading most parts of the current version of the workflow book and, while I was reading, fixed some typos and cleaned up some notation to follow the style guide described in the README.
In particular, I did a lot of fixes to the "simulated data" chapter.

The complete book compiles on my branch so should be mergable without problems.

### CODE REVIEW

**Comment by bob-carpenter** [comment]
Thanks, @paul-buerkner!

Given that we're managing this on GitHub and not through email, the lines need to be shortened to 80 characters or fewer so that we can read the diffs.  As is, this PR is pretty much impossible to review by scanning because of the multiline diffs.  And then making sure everyone's emacs or whatever doesn't slip back by catching long lines in review.

**Comment by paul-buerkner** [comment]
Thanks! I will make these changes. A lot of the text I have been editing
did originally not follow the 80 characters restriction but had multiple
100 characters at times. Accordingly diffs may still be difficult even if I
enforce 80 characters now.

Bob Carpenter <notifications@github.com> schrieb am Do., 26. März 2020,
19:16:

**Comment by paul-buerkner** [comment]
I have updated the commit so that all lines where I made changes are now limited to 80 characters.

**Comment by bob-carpenter** [comment]
Thanks.  I created an issue to do the rest:

https://github.com/jgabry/bayes-workflow-book/issues/7

**Comment by paul-buerkner** [comment]
Would it be ok, if fixed #7 as part of this PR?

**Comment by bob-carpenter** [comment]
Sure---whatever's easiest for you.

We've been taking a very casual approach to Git practices with the book.

File: introduction.Rmd [hunk 1]
```diff
@@ -524,7 +535,7 @@ step, Stan computes the gradient of the objective function, which is
 used in the "leapfrog algoritm" to determine the direction in which
 the algorithm moves (see Section 12.4 of _Bayesian Data Analysis_ and
 the "MCMC Sampling" chapter of the _Stan Reference Manual_).
-An iteration might have tens or hundreds of steps.
+A iteration might have tens or hundreds of steps.
```
Comment (mitzimorris) [inline introduction.Rmd]: "An" is correct - the "n" is required before a vowel, furthermore, the rule is phonetic, not orthographic, so if the initial consonant is silent, e.g. "hour", you also write "an hour".
https://www.grammar.com/a-vs-an-when-to-use/

File: introduction.Rmd [hunk 2]
```diff
@@ -564,7 +575,8 @@ processors.
 
 For optimization or distributional approximation, we literally want
 the different chains to converge to the same spot.  For sampling, we
-want the different chains to trace out the same distribution, which we
+want the different chains to mix well, that is, 
+to trace out the same distribution, which we
```
Comment (mitzimorris) [inline introduction.Rmd]: appreciate the clarification!

File: simulated-data.Rmd [hunk 3]
```diff
@@ -236,28 +312,74 @@ Ideally this would be done as a pre-coded function.
...
-If we can draw all parameters from the prior, that eliminates all choice from the predictive simulation.  However sometimes we can't draw from the prior (if the prior distribution is improper, that is if it has no finite integral or if it depends on the data), and other times the prior, although proper, is so weak that we would not want to draw from it because it includes highly unrealistic values.
-
-If instead we set parameters, we want to pick reasonable values, which can be done based on prior information or fit to the data.  That is, we can fit the model to the data and then perform a simulation-based check, using the data-based estimates as assumed parameter values.
-
-With hierarchical models we will often mix these approaches, choosing values for the hyperparameters and then simulating from there.
-
-Another challenge is to set the unmodeled data, for example N and x in a regression model.  These can be assigned fixed values or simulated from some distribution. In general, simulating a process requires more assumptions than are needed to fit a model.
+<!--
+Paul: Given the above content, this subsection feels a little redundant to me.
+-->
```
Comment (mitzimorris) [inline simulated-data.Rmd]: agreed - thanks for noting this via a comment.

**Comment by paul-buerkner** [comment]
Thank you @mitzimorris! I will correct the "An iteration" thing and also push a commit to enforce the 80 characters limit across the whole content (without any changes to the content itself).

**Comment by paul-buerkner** [comment]
Ok. lines should now be limited to 80 characters throughout (unless necessary for some reason).

**Comment by paul-buerkner** [comment]
Just a quick question. Is there anything that needs to be done before merging this PR? I am a little concerned that this PR will create a lot of merge conflicts once other people change anything on master as I reformatted large sections of the text due to the 80 characters limit.

**Comment by mitzimorris** [comment]
> Just a quick question. Is there anything that needs to be done before merging this PR? I am a little concerned that this PR will create a lot of merge conflicts once other people change anything on master as I reformatted large sections of the text due to the 80 characters limit.

@advaitr8 - is this going to affect you?

**Comment by bob-carpenter** [comment]
The first rule of GitHub club is to merge early and often.  So please merge unless there's some outstanding issue that's stopping you from merging it.

**Comment by advaitr8** [comment]
@mitzimorris thanks for checking with me, no this does not affect my work

**Comment by mitzimorris** [comment]
> @mitzimorris thanks for checking with me, no this does not affect my work

great - echoing Bob's comment - commit early and often.

**Pull request merged**